WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -1315,6 +1325,11 @@ private void submitSearchQuery(@NonNull String query) {
         hideSearchMessage();
         hideSearchSuggestions();
 
+        if (!NetworkUtils.isNetworkAvailable(getContext())) {
+            resetPostAdapter(mPostListType);
+            showEmptyView();
+        }
```

Comment:
Do we need to continue `submitSearchQuery` function after `showEmptyView()`? Maybe `return` from here could be better. Can you test it?

Reply:
If we add a return, we get the wrong screen shown in the image.

![image](https://user-images.githubusercontent.com/1501599/194534646-542d5d04-5cb3-4f3d-a399-3cfae14e5056.png)